### PR TITLE
Add TV licence

### PR DESCRIPTION
**Pull request by nikhilwoodruff**

### CODE REVIEW

**Comment by nikhilwoodruff** [comment]
Fixes #617

File: openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  2019-04-01: 154.50
+  2020-04-01: 157.50
+  2021-04-01: 159.00
+metadata:
+  unit: currency-GBP
+  period: year
+  name: tv_licence_fee
+  label: TV Licence fee
+  reference:
+    - title: The Communications (Television Licensing) Regulations 2004 sch. 1
```
Comment (MaxGhenis) [inline openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml]: Could we get more specific like in the US? Looks like `F4` and `F8`.

File: openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+description: Full TV licence for a colour TV, before any discounts are applied.
+values:
+  2017-04-01: 147.00
+  2018-04-01: 150.50
+  2019-04-01: 154.50
+  2020-04-01: 157.50
+  2021-04-01: 159.00
```
Comment (MaxGhenis) [inline openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml:7]: Should we uprate?

File: openfisca_uk/parameters/gov/dcms/bbc/tv_licence/tv_ownership.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+description: Percentage of households which own a TV.
+values:
+  2017-01-01: 0.9536
+  2018-01-01: 0.9541
+  2019-01-01: 0.9544
+metadata:
+  unit: household
```
Comment (MaxGhenis) [inline openfisca_uk/parameters/gov/dcms/bbc/tv_licence/tv_ownership.yaml]: Not `/1`?

File: openfisca_uk/variables/gov/dcms/bbc/tv_licence/tv_licence_discount.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        meets_pc_requirement = (
+            tv_licence.discount.aged.must_claim_pc & claims_pc
+        )
+        aged_discount = (
+            meets_pc_requirement & has_aged
+        ) * tv_licence.discount.aged.discount
+
+        # Blind discount
+        is_blind = person("is_blind", period)
+        blind_discount = (
```
Comment (MaxGhenis) [inline openfisca_uk/variables/gov/dcms/bbc/tv_licence/tv_licence_discount.py]:
```suggestion
        has_blind = household.any(is_blind)
        blind_discount = has_blind * tv_licence.discount.blind.discount
```

File: openfisca_uk/variables/gov/dcms/bbc/tv_licence/tv_licence_discount.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        tv_licence = parameters(period).gov.dcms.bbc.tv_licence
+
+        # Aged discount
+        aged = person("age", period) >= tv_licence.discount.aged.min_age
+        has_aged = household.any(aged)
+        claims_pc = add(household, period, ["pension_credit"]) > 0
+        meets_pc_requirement = (
+            tv_licence.discount.aged.must_claim_pc & claims_pc
+        )
+        aged_discount = (
```
Comment (MaxGhenis) [inline openfisca_uk/variables/gov/dcms/bbc/tv_licence/tv_licence_discount.py:27]:
```suggestion
        eligible_for_aged_discount = meets_pc_requirement & has_aged
        aged_discount = eligible_for_aged_discount * tv_licence.discount.aged.discount
```

**Review by MaxGhenis — APPROVED** [review]

File: openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml (hunk 2 above)
Comment (nikhilwoodruff) [reply openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml:7]: I believe the government has frozen the licence fee recently

Comment (nikhilwoodruff) [reply openfisca_uk/parameters/gov/dcms/bbc/tv_licence/colour.yaml:7]: https://www.gov.uk/government/news/tv-licence-fee-frozen-for-two-years

**Pull request merged**